exporter/datadogexporter/internal/metadata/internal/gohai/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// package gohai provides system information like cpu, memory
+// in a format that is expected by the datadog intake
+
+package gohai // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/metadata/internal/gohai"
```

Comment:
```suggestion
// package gohai provides system information like cpu, memory
// in a format that is expected by the Datadog intake.
package gohai // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/metadata/internal/gohai"
```
You have to put it just above the `package` declaration if you want it to be picked up by `godoc`. You can run a local instance by running `godoc -http=:` to see for yourself what the result is.